### ISSUE TITLE
GID '20' already exists Macos

### ISSUE THREAD

**Issue by Hoxolotl** [SimplyEdit/simplycode-docker#10]
After running the command to start docker, docker exits immediatedly after stating "GID '20' already exists".

I have a suspicion     --env "USER_GID=$(id -g)" \  and     --env "USER_ID=$(id -u)" \ are already assigned in macos, thus leading to a possible conflict.

```sh
docker run \
    --env "USER_GID=$(id -g)" \
    --env "USER_ID=$(id -u)" \
    --interactive \
    --rm \                                   
    --tty \                                  
    --volume "${PWD}:/var/www/www/api/data" \
    ghcr.io/simplyedit/simplycode-docker:main
    
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
There does not seem to be a "generated.html" file.
If you have not yet saved your application, this is expected.
Otherwise, this might indicate a problem with the volume mount.
All checks passed

Contents of volume /var/www/www/api/data:
	/var/www/www/api/data/
	/var/www/www/api/data/.git/
	/var/www/www/api/data/lib/
	/var/www/www/api/data/www/
	/var/www/www/api/data/.DS_Store
	/var/www/www/api/data/._.DS_Store
	/var/www/www/api/data/._Dockerfile
	/var/www/www/api/data/._docker-compose.yml
	/var/www/www/api/data/.gitignore
	/var/www/www/api/data/000-default.conf
	/var/www/www/api/data/Dockerfile
	/var/www/www/api/data/LICENSE
	/var/www/www/api/data/README.md
	/var/www/www/api/data/TODO
	/var/www/www/api/data/docker-compose.yml
Running on https://172.17.0.2/simplycode/

groupmod: GID '20' already exists
```

**Comment by Hoxolotl** [comment]
https://nickjanetakis.com/blog/running-docker-containers-as-a-non-root-user-with-a-custom-uid-and-gid might help solving the issue

**Comment by Hoxolotl** [comment]
Might already be solved after  "Change Docker workflow to use buildx #9"  is merged to main.

**Comment by Potherca** [comment]
This was indeed already resolved in #9. The docker image created for/by that branch can be tested using:

```sh
docker pull ghcr.io/simplyedit/simplycode-docker:9_merge
docker run \
    --env "USER_GID=$(id -g)" \
    --env "USER_ID=$(id -u)" \
    --interactive \
    --rm \
    --tty \
    --publish-all \
    --volume "${PWD}:/var/www/www/api/data" \
    ghcr.io/simplyedit/simplycode-docker:9_merge
```
